config/routes.rb

```diff
@@ -2,5 +2,6 @@
   get '/', to: 'locations#index'
   get 'locations/show'
   get '/notechforice', to: 'static#notech'
+  get '/branding', to: 'static#branding'
```

Comment:
It might be worthwhile to use the high-voltage gem.

Reply:
I don't know what that is or why I would use it? More words on that comment if you want to make that suggestion, please.